# Patch from upstream bug tracker:
# http://code.google.com/p/qtscriptgenerator/issues/detail?id=37
# So far (2009-06-12) this has not been applied at upstream git repo

diff -uNr generator.orig/parser/rpp/pp.h generator/parser/rpp/pp.h
--- generator.orig/parser/rpp/pp.h	2008-12-05 11:01:44.000000000 +0100
+++ generator/parser/rpp/pp.h	2008-12-13 10:42:22.000000000 +0100
@@ -30,6 +30,7 @@
 #include <cctype>

 #include <fcntl.h>
+#include <cstdio>

 #ifdef HAVE_MMAP
 #  include <sys/mman.h>
